lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -779,72 +782,65 @@ sub _failed_check {
     my $hiddens = {
        check => $check->name,
 verify_check => md5_hex($check->test_query),
-    database => $request->{database}
+    database => $request->{database},
     };
+    my @skip_keys = grep /^skip_/, keys %$request;
+    $hiddens->{@skip_keys} = $request->{@skip_keys};
 
     my $rows = [];
     while (my $row = $sth->fetchrow_hashref('NAME_lc')) {
-      my $id = $row->{$check->{id_column}};
+      my $count = 1+scalar(@$rows);
+      #TODO: A '' will fail with invalid input syntax in fix_tests - YL
```

Comment:
This comment is too terse for me. What is the problem you're trying to warn the reader for?